Cargo.toml

```diff
@@ -11,4 +11,6 @@ readme = "README.md"
 exclude = ["performance.png"]
 
 [features]
+default = ["use_std"]
 i128 = []
+use_std = []
```

Comment:
Check out [C-FEATURE](https://rust-lang-nursery.github.io/api-guidelines/naming.html#c-feature) for recommendations on how to name Cargo features.

Reply:
FWIW, I mimicked the libc crate here.

Reply:
Yeah sadly libc got this wrong, https://github.com/rust-lang/libc/issues/657. It will be fixed in their next release.